yarn-project/circuits.js/src/structs/index.ts

```diff
@@ -35,3 +35,10 @@ export * from '@aztec/foundation/eth-address';
 export * from '@aztec/foundation/fields';
 export * from '@aztec/foundation/aztec-address';
 export { FunctionSelector } from '@aztec/foundation/abi';
+
+// TODO(Kev): This is only exported so that privateKernelInit in noir-private-kernel
+// does not need to manually initialize its instance.
+// This is not great as we are exporting from tests.
+// Its okay for now and before merging into master, we should
+// remove this line.
```

Comment:
Relink comment https://github.com/AztecProtocol/aztec-packages/pull/2763#discussion_r1353737765

Reply:
should we keep this re-export?

Reply:
It was removed :D